vault/request_forwarding_rpc.go

```diff
@@ -111,9 +110,9 @@ func (c *forwardingClient) startHeartbeat() {
 				ClusterAddr: clusterAddr,
 			}
 
-			if raftStorage, ok := c.core.underlyingPhysical.(*raft.RaftBackend); ok {
-				req.RaftAppliedIndex = raftStorage.AppliedIndex()
-				req.RaftNodeID = raftStorage.NodeID()
+			if raftBackend := c.core.getRaftBackend(); raftBackend != nil {
```

Comment:
We probably only want to add these values to the reply if `!s.core.isRaftHAOnly()`

Reply:
Done in fd4ed94